### ISSUE TITLE
systemd ignores ['chef_client']['ca_cert_path'] in /etc/sysconfig/chef-client when set on rhel

### ISSUE THREAD

**Issue by bcg62** [chef-cookbooks/chef-client#563]
`systemd: Ignoring invalid environment assignment 'export SSL_CERT_FILE=/etc/pki/tls/certs/ca-bundle.crt': /etc/sysconfig/chef-client`

I suspect the export needs to be removed

https://github.com/chef-cookbooks/chef-client/blob/master/templates/default/redhat/sysconfig/chef-client.erb#L16

**Comment by bcg62** [comment]
Doing it the same as suse:

```
<% if node["chef_client"]["ca_cert_path"] %>
SSL_CERT_FILE="<%= node["chef_client"]["ca_cert_path"] %>"
export SSL_CERT_FILE
<% end %>
```
https://github.com/chef-cookbooks/chef-client/blob/master/templates/default/suse/sysconfig/chef-client.erb

seems to correct the issue with no systemd warnings.

tested on centos 7.4.1708

**Comment by tas50** [comment]
Fixed in 10.0.2
